src/components/ListItems/ListItems.js

```diff
@@ -2,10 +2,20 @@ import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faTrash } from '@fortawesome/free-solid-svg-icons';
 
+import styles from './styles.module.css';
+
 const ListItems = ({ entries, deleteItem }) => (
   <ul>
     {entries.map(({ key, text }) => (
-      <li key={key}>{text} <FontAwesomeIcon icon={faTrash} onClick={() => deleteItem(key)} /></li>
+      <li key={key}>
+        {text}
+        {' '}
+        <FontAwesomeIcon
+          icon={faTrash}
+          onClick={() => deleteItem(key)}
+          className={styles.deleteIcon}
```

Comment:
Looks good and makes sense. Is the intention to end up with a mix of global and local css scopes, or only local? Just trying to get a better picture of how development of the stylesheets will work.

[react-css-modules](https://github.com/gajus/react-css-modules) looks like it might provide some advantages with CSS Modules - in particular identifying if there is an undefined className and throwing up a warning.

Reply:
I think in the end almost all of it will be local to specific components. Specific things might be global, but they'll be the exceptions. Out of the stuff we currently have in index.css I think we only need to keep the `body` part global:

https://github.com/mount-joy/thelist/blob/843fb82ce89d233af7de5de43b2d327e96729bc0/src/style/index.css#L1-L23

Reply:
Interesting find with react-css-modules, there's an article about it here that I just read, some interesting points:
https://medium.com/htc-research-engineering-blog/css-modules-with-react-50a20ca39088